numpy/core/src/common/npy_cpu_features.h

```diff
@@ -1,8 +1,8 @@
 #ifndef _NPY_CPU_FEATURES_H_
 #define _NPY_CPU_FEATURES_H_
 
-#include "numpy/numpyconfig.h" // for NPY_VISIBILITY_HIDDEN
 #include <Python.h> // for PyObject
+#include "numpy/numpyconfig.h" // for NPY_VISIBILITY_HIDDEN
```

Comment:
I am curious why the shift in order is needed, usually `Python.h` comes first

Reply:
to suppress warning 'declaration of 'struct timespec*', this compiler warnning raised when `math.h` get included before
`Python.h`.

Reply:
Do you have some more info about this you can link to?